aea/protocols/generator/common.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
```

Comment:
This module has code common to all generator modules. Currently only one function and one constant.